src/collection-proxy.js

```diff
@@ -220,6 +220,16 @@ class CollectionProxy {
     )
   }
 
+  /**
+   * Creates an array of unique values that are included in both given array
+   * @param {Array} items

```

Comment:
Can you please add an empty line between the comment and the `@param`?

```suggestion
   *
   * @param {Array} items
```